hub/powertoys/install.md

```diff
@@ -132,6 +132,6 @@ After successfully installing PowerToys, an overview window will display with in
 
 ## Updates
 
-PowerToys uses an auto-updater that checks for new versions when the app is running. If enabled, a toast notification will appear when an update is available. Updates can also be checked for manually from the PowerToys Settings, under the General page.
+PowerToys uses an auto-updater that checks for new versions when the app is running. If enabled, a toast notification will appear when an update is available. Updates can also be checked for manually from the PowerToys Settings, on the **General** page.
```

Comment:
In some previous docs in this PR, I noticed that the page name in settings was not bold. I'll have to go back and find those, if you want to make them consistent. Checking for those could probably be another PR though.